python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -463,6 +463,16 @@ def _db_connect(self):
             with contextlib.closing(connection.cursor()) as cursor:
                 yield (connection, cursor)
 
+    def _compute_sys_path(self):
```

Comment:
It's actually dependent on the state of the sgtk instance. This is not something the manager can give us.